code/ATMOSPHERICS/pipes.dm

```diff
@@ -27,10 +27,8 @@
 		var/datum/pipeline/pipeline = parent
 		var/list/update_later = list()
 		for(var/obj/machinery/atmospherics/pipe in pipeline.members)
-			pipe.color = mass_colour
-			if(!pipe.can_be_coloured)
-				pipe.default_colour = mass_colour
-				update_later += pipe
```

Comment:
By the way, what the fuck was whoever wrote this thinking?

Reply:
:woman_shrugging: